Testsuite NDF

```
Test go.sia.tech/explored/persist/sqlite/TestScan/online_v1_host#02 output
  Error: scan_test.go:427: SuccessfulInteractions expected != got, diff:   uint64(
          - 	3,
          + 	2,
            )
  === RUN   TestScan/online_v1_host#02
      scan_test.go:427: SuccessfulInteractions expected != got, diff:   uint64(
          - 	3,
          + 	2,
            )
      --- FAIL: TestScan/online_v1_host#02 (0.00s)
```

see https://github.com/SiaFoundation/explored/actions/runs/18347794266/job/52259117818?pr=280

--- a/persist/sqlite/scan_test.go
+++ b/persist/sqlite/scan_test.go
@@ -330,8 +330,8 @@ func TestScan(t *testing.T) {
 
 	cfg := config.Scanner{
 		NumThreads:          100,
-		ScanTimeout:         100 * time.Millisecond,
-		ScanFrequency:       100 * time.Millisecond,
+		ScanTimeout:         500 * time.Millisecond,
+		ScanFrequency:       25 * time.Millisecond,
 		ScanInterval:        3 * time.Hour,
 		MinLastAnnouncement: 90 * 24 * time.Hour,
 	}

